localstack/services/apigateway/templates.py

```diff
@@ -104,6 +115,9 @@ def escapeJavaScript(self, obj: Any) -> str:
             return str(obj).lower()
         return str(obj)
 
+    def parseJson(self, s):
+        return AttributeDict(json.loads(s))
```

Comment:
I guess the result of `json.loads` could also be a `list` or other type. Let's better check if the result is a `dict`, and only then wrap it in an `AttributeDict`.. 👍

Reply:
Yep, that's right! 

For the following string:

`{"errorMessage":"{\"key1\":\"var1\",\"key2\":{\"arr\":[1,2,3]}}"}`

and the VTL template
```
#set ($errorMessageObj = $util.parseJson($input.path('$.errorMessage')))
{
   "errorMessageObjKey2ArrVal" : $errorMessageObj.key2.arr[0]
}
```

should yield,

```
{
   "errorMessageObjKey2ArrVal" : 1
}
```

Reply:
I reworked this a bit to apply to all keys.